configuration/environment_variables.rst

```diff
@@ -788,3 +788,87 @@ configuration::
 
 .. _`SetEnv`: http://httpd.apache.org/docs/current/env.html
 .. _`fastcgi_param`: http://nginx.org/en/docs/http/ngx_http_fastcgi_module.html#fastcgi_param
+
+Debugging Variables
+---------------------------
+
+.. versionadded:: 4.3
+
+    Environment variables debugging was introduced in Symfony 4.3.
+
+You can find out which environment variables are used by the container using the
+console. To show all variables and their values (default and real), run:
+
+.. code-block:: terminal
+
+    $ php bin/console debug:container --env-vars
+
+    ---------------- ----------------- ---------------------------------------------
+     Name             Default value     Real value
+    ---------------- ----------------- ---------------------------------------------
+     APP_SECRET       n/a               "471a62e2d601a8952deb186e44186cb3"
+     FOO              "[1, "2.5", 3]"   n/a
+     BAR              null              n/a
+    ---------------- ----------------- ---------------------------------------------
+
+    // Note real values might be different between web and CLI.
```

Comment:
maybe a note detailing default vs real values?

Reply:
Is it really necessary as the default processor is described just ahead?

Reply:
I've added a small note for this